src/main/webapp/js/userMapData.json

```diff
@@ -95,7 +103,8 @@
       "Bahrain Polytechnic"
     ],
     "Botswana": [
-      "Alpha"
+      "Alpha",
+      "University of Botswana"
     ],
     "Brazil": [
       "AEDB",
```

Comment:
A quick comment (not related to your changes, but can be fixed in this PR): this line is same as the next line. Keep this and remove the next line.